docs/changelog/78968.yaml

```diff
@@ -0,0 +1,20 @@
+pr: 78968
+summary: HTTP Status code has changed for the Cluster Health API in case of a server timeout
```

Comment:
This duplicates the entry in `docs/reference/migration/migrate_8_0/cluster.asciidoc` - are you sure both are needed?

Reply:
I'm not sure, but I got a warning from @elasticmachine that I need to create a changelog entry when I add a `breaking` label to the PR.

Reply:
Yes I've not seen this before, I suggest checking with delivery or docs folks what to do about this.